scm_staging/rabbit_listener.py

```diff
@@ -144,6 +146,10 @@ class RequestCommentPayload(RequestChangedPayload):
 
 def _process_message(routing_key: str, body: str) -> None:
     """Smoke test for the message payload models"""
+    if re.match("opensuse.obs.request", routing_key):
+        with open("/tmp/msg_debug_log", mode="a") as f:
```

Comment:
This will be running in a container with cwd bind mounted. `/tmp/` will get lost on container restart, hence that would be a bad location.
```suggestion
    # FIXME: remove this once we find out why the bot doesn't merge some PRs
    if re.match(r"^opensuse\.obs\.request", routing_key):
        with open("./msg_debug.log", mode="a") as f:
```